doc/DEMO.md

```diff
@@ -0,0 +1,629 @@
+cd ..
+```
+
+Now start the project:
+```
+cargo run -- --conf cosigner_2_config.toml
+```
+
+It will print a Bitcoin public key. Save it in your notes, then kill the cosignerd with `CTRL-C`.
+
+Your notes should now look like this:
+```
+stakeholder1: *xpriv* *xpub*
+stakeholder2: *xpriv* *xpub*
+manager1: *xpriv* *xpub*
+cosigner1: *bitcoin pubkey*
+cosigner2: *bitcoin pubkey*
+```
+
+Go back to the parent directory for the next step
+```
+cd ..
+```
+### 4. Creating the revaultd configuration
+Cd into the project:
+```
+cd revaultd
+```
+
+Build it:
+```
+cargo build --release
+```
+
+Now we need to setup `revaultd` three times, one for each entity.
+
+#### Stakeholder 1
+Let's create a directory for storing revaultd data:
+```
+mkdir stake_1_data
+```
+
+`revaultd` needs a configuration file - you can find [here](../contrib/config_regtest.toml) an example of configuration.
+The default path for the configuration is `~/.revault/revault.toml`, but for this example we'll use `./stake_1_config.toml`:
+```
+cp contrib/config_regtest.toml stake_1_config.toml
+```
+
+We'll need to modify the config a bit to fit our example:
+- Update the `data_dir`: this is the directory where all the data will be saved. We'll use `./stake_1_data
+- Update the `coordinator_host`, `coordinator_noise_key` with the data obtained in step 2
+- Update the `stakeholders_xpubs`: it will contain all the stakeholder xpubs created in the ceremony
+
+```
+stakeholder_xpubs = [
+    *first stakeholder xpub*,
+    *second stakeholder xpub*,
+]
+```
+Please make sure that the keys are in the right order, the first as first and the second as second. Not doing so could lead to tricky bugs.
+- Update the `managers_xpubs`: it will contain all the manager xpubs created in the ceremony:
+
+```
+manager_xpubs = [
+    *first manager xpub*,
+]
+```
+
+- Update the `cosigner_keys`: it will contain all the cosigner keys created in step 3
+
+```
+cosigner_keys = [
+    *first cosigner key*,
+    *second cosigner key*,
+]
+```
+Please make sure that the keys are in the right order, the first as first and the second as second. Not doing so could lead to tricky bugs.
+
+- Update the `stakeholder_config`: update `xpub` to match the first stakeholder's xpub; leave the watchtowers section as it is, it's not used (yet!)
+
+- Update the `bitcoind_config` `addr` to the address of your bitcoin RPC. In regtest defaults to `127.0.0.1:18832`. The one contained in the regtest config is the correct one if you're using the regtest manager script, as outlined below
+
+#### Stakeholder 2
+Let's create a directory for storing revaultd data:
+```
+mkdir stake_2_data
+```
+
+`revaultd` needs a configuration file - you can find [here](../contrib/config_regtest.toml) an example of configuration.
+The default path for the configuration is `~/.revault/revault.toml`, but for this example we'll use `./stake_2_config.toml`:
+
+```
+cp contrib/config_regtest.toml stake_2_config.toml
+```
```

Comment:
Trying this for real what i did is just `cp stake_1_config.toml stake_2_config.toml` and changing the stakeholder xpub